### ISSUE TITLE
Use autofocus HTML5 property wherever we want something to autofocus

### ISSUE THREAD

**Issue by mitar** [peerlibrary/peerlibrary#406]
Use `autofocus` HTML5 property wherever we want something to autofocus when rendered. We can still use manual JavaScript focus as failback if we see that it is needed, but we should still mark them with `autofocus`.
